x-pack/plugins/data_quality/server/routes/get_index_stats.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { IRouter } from '@kbn/core/server';
+import { transformError, buildResponse } from '@kbn/securitysolution-es-utils';
+
+import { fetchStats } from '../lib';
+import { GET_INDEX_STATS } from '../../common/constants';
+
+export const getIndexStatsRoute = (router: IRouter) => {
+  router.get(
+    {
+      path: GET_INDEX_STATS,
+      validate: {
+        params: schema.object({
+          index_name: schema.string(),
```

Comment:
Consider renaming the `request.params.index_name` parameter to `request.params.pattern`, because the Data Quality Dashboard will provide (as input to this route) a pattern like `logs-*`, to enumerate all the indexes that match the specified pattern.